test/Constraints/optional.swift

```diff
@@ -436,9 +436,15 @@ func sr_12309() {
 
   _ = S(foo: nil!) // expected-error {{'nil' literal cannot be force unwrapped}}
   _ = nil! // expected-error {{'nil' literal cannot be force unwrapped}}
+  _ = (nil!) // expected-error {{'nil' literal cannot be force unwrapped}}
+  _ = (nil)! // expected-error {{'nil' literal cannot be force unwrapped}}
```

Comment:
Question... @xedin should this also have a multi paren with `!` test case e.g. `((((nil)))!` maybe?

Reply:
I think it's a good idea to maybe include `((nil))!` but just that.

Reply:
@HassanElDesouky mind taking care of that?

Reply:
Not a problem. I’ll take care of this.

Reply:
Thank you!